src/raw/_Raw.jsx

```diff
@@ -30,6 +31,11 @@ const propTypes = {
   }).isRequired,
 };
 
+const promptProviderValue = {
```

Comment:
This should be stored in a ref or in state so it's not continuously rendered as a new object value: https://reactjs.org/docs/context.html#caveats

Reply:
Since it's declared outside of the component, it will always have a consistent reference. So this should be alright.

Reply:
This won't get rendered again because it's defined outside of the component.